net/zerotier/src/opnsense/mvc/app/controllers/OPNsense/Zerotier/Api/ZerotierController.php

```diff
@@ -45,12 +45,26 @@ public function getAction()
     {
         $result = array();
         if ($this->request->isGet()) {
-            $mdlZerotier = new Zerotier();
+            $mdlZerotier = $this->getModel();
```

Comment:
you can also keep the old version. getModel does exactly this with reflection using `$internalModelClass`.

Reply:
Hi, I changed it to getModel to be consistent throughout the codebase. No other reason. If you recommend doing a new each time, then I can put that in again.